docs/source/dev/PlantUML.rst

```diff
@@ -0,0 +1,28 @@
+.. _PlantUML_ŕeference:
+
+PlantUML
+========
+
+`PlantUML <https://plantuml.com/>`_ is a java tool generating UML, and similar, diagrams from text files and has been used to generate UML diagrams in our documentation.
+For each UML diagram generated using PlantUML we also provide the source file ``<figure_name>.txt``.
+
+Install PlantUML
+----------------
+To install PlantUMl you need to
+- install a compatible java version
+- download ``plantUML.jar`` from the `PlantUML website <https://plantuml.com/download>`_ and place it in a folder of your choice.
+
+Updating Figures
+----------------
+
+To update a figure edit the ``<figure_name>.txt`` file and regenerate the figure by running the following command in the folder containing the ``plantUML.jar file`` execute the following command.
+
+.. code:: bash
+
+  java -DPLANTUML_LIMIT_SIZE=8192 -jar plantuml.jar
+
+This will open a allowing you to choose a directory in which PlantUML will try to regenerate all figures from the contained source files.
```

Comment:
```suggestion
This will open a ??? allowing you to choose a directory in which PlantUML will try to regenerate all figures from the contained source files.
```

Reply:
fixed